tensorflow/core/common_runtime/threadpool_device.cc

```diff
@@ -49,6 +50,9 @@ ThreadPoolDevice::ThreadPoolDevice(const SessionOptions& options,
       allocator_(allocator),
       scoped_allocator_mgr_(new ScopedAllocatorMgr(name)) {
 #ifdef INTEL_MKL
+  // Early return when MKL is disabled
+  if (DisableMKL())
+    return;
```

Comment:
Please run `clang-format` on all files. I believe this `if` would be formatted into one line. (No newline before`return`.)